test/integration/scheduler_perf/scheduler_perf_test.go

```diff
@@ -72,11 +72,21 @@ type nodeCase struct {
 type podCase struct {
 	Num                               int
 	PodTemplatePath                   *string
-	ReplicationControllerName         *string
 	PersistentVolumeTemplatePath      *string
 	PersistentVolumeClaimTemplatePath *string
 }
 
+type simpleTestCases struct {
```

Comment:
Can you add a comment such as 
"simpleTestCases defines a set of test cases that share the same template (node spec, pod spec, etc) with testParams(e.g., NumNodes) being overriden. This provides a convenient way to define multiple tests with various sizes."